### DISCUSSION TITLE
How can I make active and inactive windows more visually distinct?

### DISCUSSION THREAD

**Discussion by tneisinger** [LunarVim/LunarVim#3674] [Q&A] [answered]
I'm trying to configure LunarVim so that the active window stands out very strongly from the inactive windows.  In vanilla neovim, this works for me:

```
vim.cmd([[
hi def Dim cterm=none ctermbg=none ctermfg=242
function! s:DimInactiveWindow()
    syntax region Dim start='' end='$$$end$$$'
endfunction
function! s:UndimActiveWindow()
    ownsyntax
endfunction
autocmd WinEnter * call s:UndimActiveWindow()
autocmd BufEnter * call s:UndimActiveWindow()
autocmd WinLeave * call s:DimInactiveWindow()
]])
```

That will make all inactive windows gray scale, leaving the active window in color.  But that doesn't work in the LunarVim config file.  Any suggestions?  Thanks!

**Comment by mamachanko** [comment]
I am not 💯 but I think that LunarVim sets up [statusline](https://github.com/akinsho/bufferline.nvim). Maybe you can find a solution in its docs.

**Comment by tneisinger** [comment, marked as answer]
I ended up installing the [shade](https://github.com/sunjon/Shade.nvim) plugin by adding this to my config:

```
lvim.plugins = {
  {
    "sunjon/shade.nvim",
    config = function()
      require("shade").setup({
        overlay_opacity = 30,
        opacity_step = 2,
        keys = {
          brightness_up   = '<C-Up>',
          brightness_down = '<C-Down>',
          toggle          = '<Leader>x'
        }
      })
    end,
  }
}
```
